msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/PrivateMessagesTool.java

```diff
@@ -439,7 +441,19 @@ public void initializePrivateMessageArea() {
       forum=pf;
       activatePvtMsg = (Boolean.TRUE.equals(area.getEnabled())) ? SET_AS_YES : SET_AS_NO;
       sendToEmail = area.getSendToEmail() + "";
-      forwardPvtMsg = (Boolean.TRUE.equals(pf.getAutoForward())) ? SET_AS_YES : SET_AS_NO;
+
+      switch (pf.getAutoForward()) {
+      case PrivateForumImpl.AUTO_FOWARD_YES:
```

Comment:
Thanks for the review, fix indentation here.

Reply:
Fixed this one and others minor fixed